src/sentry/static/sentry/app/components/onboardingWizard/getOnboardingTasks.tsx

```diff
@@ -134,7 +134,7 @@ export default function getOnboardingTasks(
       prereq: [1],
       actionType: 'app',
       location: `/settings/${organization.slug}/projects/:projectId/alerts/`,
-      display: false,
+      display: true,
```

Comment:
Should this be here?

Reply:
Yes, this enables the onboarding task on the frontend.